style.css

```diff
@@ -239,6 +239,7 @@
 
 .partners {
   display: grid;
+  margin-left: 120px;
 }
```

Comment:
- [ ] While you fix spacing in desktop version, it is affect in mobile version, add `margin-left: 120px;` for class `partners` inside media query to affect only in desktop version.
> your design in mobile version
![image](https://user-images.githubusercontent.com/53402600/175786550-be847adf-0739-4460-b33a-19dce49a60dd.png)